solidity/random-beacon/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@keep-network/random-beacon-v2",
+  "version": "0.0.1-dev",
+  "description": "random beacon v2",
+  "scripts": {
+    "build": "hardhat compile",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "dependencies": {
+    "@openzeppelin/contracts": "^4.3.2",
+    "@tenderly/hardhat-tenderly": "^1.0.12",
```

Comment:
Shouldn't this be a dev dependency?

Reply:
Also worked in dev dependency.  I thought it should belong to dependency because we're using that during deployment 168397b